yarn-project/simulator/src/avm/avm_context.ts

```diff
@@ -43,13 +44,13 @@ export class AvmContext {
     calldata: Fr[],
     allocatedGas: Gas,
     callType: 'CALL' | 'STATICCALL',
-    functionSelector: FunctionSelector = FunctionSelector.empty(),
+    fnName: string,
```

Comment:
See my comment on avm_simulator.ts, this doesn't feel right.

Reply:
Removed here as well.